apps/site/assets/ts/components/Modal.tsx

```diff
@@ -61,7 +69,8 @@ const ModalContent = ({
   ariaLabel,
   focusElementId,
   bodyPadding,
-  scrollBarPadding
+  scrollBarPadding,
+  className
```

Comment:
Probably should default this to an empty string if not provided, you can see the `undefined`s in the snapshot